qiita_pet/handlers/study_handlers/tests/test_artifact.py

```diff
@@ -140,6 +140,33 @@ def test_delete_artifact(self):
         wait_for_prep_information_job(1)
 
 
+class ArtifactGetSamplesTest(TestHandlerBase):
+    def test_get(self):
+        response = self.get('/artifact/samples/', {'ids[]': [4, 5]})
+        self.assertEqual(response.code, 200)
+        exp = (
+            '{"status": "success", "msg": "", "data": {"4": ['
```

Comment:
> we are certain it will not change

As far as I know, dictionary keys in python are not guaranteed to remain in the same order, so `artifact_get_prep_req` could return the exact same object but formatted differently, think of:

`'{"status": "success", "msg": "", "data": { ...'`

vs

`'{"msg": "", "status": "success", "data": { ...'`

While the object is the same, the test would fail, this wouldn't happen if the objects were loaded as dictionaries.

I would load the body of the response using `from json import loads` and store this exp either as a python dictionary or load it as well with `loads`.